packages/tailwindcss/src/utilities.test.ts

```diff
@@ -16748,11 +16748,11 @@ describe('custom utilities', () => {
     expect(optimizeCss(compiled).trim()).toMatchInlineSnapshot(`
       "@layer utilities {
         .text-sm {
-          font-size: var(--text-sm, .875rem);
-          line-height: var(--tw-leading, var(--text-sm--line-height, 1.25rem));
           font-size: var(--text-sm, .8755rem);
           line-height: var(--text-sm--line-height, 1.255rem);
           text-rendering: optimizeLegibility;
+          font-size: var(--text-sm, .875rem);
+          line-height: var(--tw-leading, var(--text-sm--line-height, 1.25rem));
```

Comment:
This diff is weird but that's because of lightning flatting. What happens ehre is the custom new `text-sm` utility is now rendered before the core `text-sm`. The new prop ` text-rendering: optimizeLegibility;` is still added, the rest is intended behavior here since you can't _overwrite_ core utilities.